Project/package.json

```diff
@@ -3,32 +3,32 @@
   "version": "0.3.0",
   "private": true,
   "dependencies": {
-    "@fluentui/react": "^7.137.1",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0",
-    "@azure/communication-common": "1.0.0-beta.3",
-    "@azure/communication-administration": "1.0.0-beta.3",
-    "@azure/communication-calling": "1.0.0-beta.3",
-    "@azure/logger": "1.0.0"
+    "@azure/communication-administration": "1.0.0-beta.4",
+    "@azure/communication-calling": "1.0.0-beta.4",
+    "@azure/communication-common": "1.0.0-beta.4",
+    "@azure/logger": "1.0.1",
+    "@fluentui/react": "^7.158.1",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0"
   },
   "scripts": {
     "start": "webpack-dev-server --port 5000 --mode development",
     "build": "webpack --mode development"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.7",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.8.3",
-    "@babel/preset-env": "^7.8.7",
-    "@babel/preset-react": "^7.8.3",
-    "@babel/runtime": "^7.8.7",
-    "babel-loader": "^8.0.6",
+    "@babel/core": "^7.12.13",
+    "@babel/plugin-proposal-class-properties": "^7.12.13",
+    "@babel/plugin-transform-runtime": "^7.12.15",
+    "@babel/preset-env": "^7.12.13",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/runtime": "^7.12.13",
+    "babel-loader": "^8.2.2",
     "css-loader": "^3.4.2",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
-    "style-loader": "^1.1.3",
-    "webpack": "^4.42.0",
+    "style-loader": "^1.3.0",
+    "webpack": "^4.46.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3"
+    "webpack-dev-server": "^3.11.2"
```

Comment:
Iv had some issues with combinations of these versions before, why update them? Just to use newer ones?

Please rm -rf node_modules, then run npm install + build and test to make sure all works fine.

Reply:
I thought npm update would update communication dependencies but it updated webpack.
Sorry, my bad